core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -410,38 +408,24 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.isOnTerminalCard = function() {
-            return $scope.activeCard &&
-              ExplorationPlayerStateService.isCurrentStateTerminal();
+            return (
+              $scope.displayedCard && $scope.displayedCard.isCardTerminal());
```

Comment:
Maybe let's rename it to isTerminal() -- no need to repeat "card".

Reply:
Done.